scripts/hud/customizer.ts

```diff
@@ -0,0 +1,644 @@
+// I'm trying something new here and using a namespace to escape the namespace pollution nightmare currently have
+// In short, TypeScript seems to put stuff that doesn't include `import`/`export`s in the global namespace,
+// and using those is impossible since pure V8 doesn't have an import/module system.
+
+// The below `Axis` enum is breaking compiles because we have some fucking JSDOC (!) typedef in linegraph.js
+// that overlaps with this one. So, I'm using a namespace (which just gets compiled to an IIFE for scope isolation)
+// and not using a static class - that was essentially just playing the role of an IIFE anyway!
+
+// Not sure if this is what we'll stick with it, but seems good for now.
+
+// Sasha: had to change some types from Panel to GenericPanel
```

Comment:
Prefer to avoid tracking changes in source code, we tend to keep our commit history very organised so Git's enough for tracking changes. I know this is for a change you fixuped into my commit, but I'd just add something to the commit message/leave it out completely.

Reply:
yeah I just added this for the review